packages/react/src/global-alert/docs/overview.mdx

```diff
@@ -57,3 +61,19 @@ Use the `onDismiss` prop to make the alert dismissable.
 	</Text>
 </GlobalAlert>
 ```
+
+## Infomation alert
+
+An `info` version of GlobalAlert is also available, which can be used to display system-wide information messages which are not negative in nature. This could include a general announcement about a feature or policy.
```

Comment:
```suggestion
An `info` version of Global alert is also available, which can be used to display system-wide information messages which are not negative in nature. This could include a general announcement about a feature or policy.
```